go-fuzz/hub.go

```diff
@@ -116,6 +118,24 @@ func newHub(metadata MetaData) *Hub {
 			ro.intLits = append(ro.intLits, []byte(lit.Val))
 		}
 	}
+	if *flagDict != "" {
+		ro.strLits = nil // Discard existing tokens
+		fileName := *flagDict
+		log.Printf("reading token list from \"%v\"\n", fileName)
+		dictData, err := ioutil.ReadFile(fileName)
+		if err != nil {
+			log.Fatalf("could not read tokens from \"%v\": %v\n", fileName, err)
```

Comment:
Use %q instead of "%v".

Reply:
Also don't add `\n` at the end of Fatalf/Printf, it's added automatically and the rest of the code base does not add them manually.